browser/src/Config.ts

```diff
@@ -79,23 +128,23 @@ class Config extends EventEmitter {
         "editor.cursorColumnOpacity": 0.1,
     }
 
-    private MacConfig: any = {
+    private MacConfig: Partial<IConfigValues> = {
         "editor.fontFamily": "Monaco",
     }
 
-    private WindowsConfig: any = {
+    private WindowsConfig: Partial<IConfigValues> = {
         "editor.fontFamily": "Consolas",
     }
 
-    private LinuxConfig: any = {
+    private LinuxConfig: Partial<IConfigValues> = {
```

Comment:
Cool, I didn't know about `Partial` - that's really helpful for this scenario. I think it would be useful for some of the React typings, especially how `connect` works with `mapStateToProps` and `mapDispatchToProps`. Nice!